addons/l10n_ee/data/account_fiscal_position_template_data.xml

```diff
@@ -163,6 +177,21 @@
         <field name="tax_dest_id" ref="l10n_ee_vat_in_0_eu_g_22"/>
     </record>
 
+    <!-- Purchase of Goods 22% -> Purchase of Goods in EU 0% IC -->
+    <record id="afptt_eu_ic_19" model="account.fiscal.position.tax.template">
```

Comment:
I must admit that the xml_ids of these fp template are quite horrible.
But that's the situation before you ^^